app/views/qsets/_modal_question.erb

```diff
@@ -8,12 +8,12 @@
         <!-- Displays the question at the top of the modal. Javascript finds the "modal-title" class and populates it with the question contents. -->
         <h4 class="modal-title"></h4>
       </div>
-      <div class="modal-body">
+      <div class="modal-body center">
 
         <!-- Creates the text box for the user to input their answer. The submit button is only able to be clicked once the user has some content in the text box. -->
         <form accept-charset="UTF-8"
               action="/questions/question['id']/answers"
-              class="new_answer"
+              class="new_answer center"
```

Comment:
I think these `center` classes are making the <textarea> text centered as well, which looks different from the modal. Can you make the UX similar for both?